nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/ui/web/auth/NuxeoAuthenticationFilter.java

```diff
@@ -299,14 +299,21 @@ protected Principal doAuthenticate(CachableUserIdentificationInfo cachableUserId
 
             logAuthenticationAttempt(cachableUserIdent.getUserInfo(), true);
         } catch (LoginException e) {
-            log.info("Login failed for " + cachableUserIdent.getUserInfo().getUserName());
+            log.info(String.format("Login failed for %s on request %s", cachableUserIdent.getUserInfo().getUserName(),
+                    httpRequest.getRequestURI()));
+            log.debug(e, e);
             logAuthenticationAttempt(cachableUserIdent.getUserInfo(), false);
             Throwable cause = e.getCause();
             if (cause instanceof DirectoryException) {
                 Throwable rootCause = ExceptionUtils.getRootCause(cause);
                 if (rootCause instanceof NamingException
                         && rootCause.getMessage().contains("LDAP response read timed out")
                         || rootCause instanceof SocketException) {
+                    log.debug(String.format(
```

Comment:
Could you add a `isDebugEnabled()` ? To prevent possible CPU consumption in case of attack.